script.js

```diff
@@ -1,4 +1,114 @@
+//Project 2: Beat That (Base+Comfortable)
+
+var rollDice = "ROLL DICE";
+var chooseSequence = "CHOOSE SEQUENCE";
+var gameMode = rollDice;
+var player1;
+var player2;
+var currentPlayer = 1;
+var player1Result = []; //player1'sDiceValues
+var player2Result = []; //player2'sDiceValues
+
+var diceRoll = function () {
+  return Math.floor(Math.random() * 6 + 1);
+};
+
+var obtainDiceRolls = function () {
+  var newDiceRolls = [diceRoll(), diceRoll()];
+
+  if (currentPlayer === 1) {
+    player1Result = newDiceRolls;
+  } else {
+    player2Result = newDiceRolls;
+  }
+
+  return newDiceRolls;
+};
+
+var concatenate2Numbers = function (randNum1, randNum2) {
+  return Number(String(randNum1) + String(randNum2));
+};
+
+var obtainPlayerNumber = function (firstNumeralIndex) {
+  var diceArray;
+  if (currentPlayer === 1) {
+    diceArray = player1Result;
+  } else {
+    diceArray = player2Result;
+  }
+  var playerNum;
+  // If the chosen first numeral index is 1, create player number starting with 1st dice
+  if (firstNumeralIndex === 1) {
+    playerNum = concatenate2Numbers(diceArray[0], diceArray[1]);
+  }
+  // Otherwise, create player number starting with 2nd dice
+  else {
+    playerNum = concatenate2Numbers(diceArray[1], diceArray[0]);
+  }
+
+  if (currentPlayer === 1) {
+    player1 = playerNum;
+  } else {
+    player2 = playerNum;
+  }
+
+  //Return generated player number
+  return playerNum;
+};
+
+//Determine who wins
+var determineWinner = function () {
+  if (player1 > player2) {
+    return 1;
+  }
+  return 2;
+};
```

Comment:
What about a draw scenario? For this code it seems like player2 automatically wins during a draw